frontend/pages/shopping-lists/_id.vue

```diff
@@ -1025,7 +1064,7 @@ export default defineComponent({
       }
 
       // update current user
-      allUsers.value = data.items.sort((a, b) => ((a.fullName || "") < (b.fullName || "") ? -1 : 1));
+      allUsers.value = data.sort((a, b) => ((a.fullName || "") < (b.fullName || "") ? -1 : 1));
       currentUserId.value = shoppingList.value?.userId;
```

Comment:
Looks mostly good. Only thing is this, which destroys the settings window. 

```suggestion
      allUsers.value = data.items.sort((a, b) => ((a.fullName || "") < (b.fullName || "") ? -1 : 1));
```

Reply:
Sorry about that.  It should be corrected now.